yarn.lock

```diff
@@ -1138,6 +1159,19 @@
   resolved "https://registry.yarnpkg.com/@jridgewell/sourcemap-codec/-/sourcemap-codec-1.4.14.tgz#add4c98d341472a289190b424efbdb096991bb24"
   integrity sha512-XPSJHWmi394fuUuzDnGz1wiKqWfo1yXecHQMRf2l6hztTO+nPru658AyDngaBe7isIxEkRsPR3FZh+s7iVa4Uw==
 
+"@jridgewell/sourcemap-codec@^1.4.14":
+  version "1.4.15"
+  resolved "https://registry.yarnpkg.com/@jridgewell/sourcemap-codec/-/sourcemap-codec-1.4.15.tgz#d7c6e6755c78567a951e04ab52ef0fd26de59f32"
+  integrity sha512-eF2rxCRulEKXHTRiDrDy6erMYWqNw4LPdQ8UQA4huuxaQsVeRPFl2oM8oDGxMFhJUWZf9McpLtJasDDZb/Bpeg==
+
+"@jridgewell/trace-mapping@^0.3.17":
```

Comment:
Could you run [yarn-deduplicate](https://github.com/scinos/yarn-deduplicate) on this? It looks like this could be deduped with the entry below.

Reply:
@aduth  I ran yarn-deduplicate. The entire yarn.lock file got modified so I discarded the changes as a duplicate for this packages does not appear to exist. (It only modified the order of integrity, resolved, and version for every package. I did not see @jridgewell/trace-mapping@^0.3.17 twice. The working tree has the same number of line numbers for the file- so no de-duping occurred.)